src/styles/02-molecules/input/input.scss

```diff
@@ -0,0 +1,26 @@
+.input {
+  
+  &__label {
+    @include space-inline-xs;
+    display: inline-block;
```

Comment:
Stylistic and we should have a formatter for it, but please add a newline after `@include`s.